package.json

```diff
@@ -73,6 +73,7 @@
     "@typescript-eslint/parser": "^5.12.1",
     "abort-controller": "^3.0.0",
     "abortcontroller-polyfill": "^1.7.3",
+    "arg": "^5.0.1",
```

Comment:
Why do we need this dependency in the root? I think we can remove it? It looks like meow was only ever being used in remix-dev.